llvm/lib/ProfileData/RawMemProfReader.cpp

```diff
@@ -344,7 +344,7 @@ Error RawMemProfReader::initialize(std::unique_ptr<MemoryBuffer> DataBuffer) {
       Object, std::move(Context), /*UntagAddresses=*/false);
   if (!SOFOr)
     return report(SOFOr.takeError(), FileName);
-  Symbolizer = std::move(SOFOr.get());
+  auto Symbolizer = std::move(SOFOr.get());
```

Comment:
I wonder if it makes a difference to the peak memory usage if we move L343-L347 after the readRawRawProfile call. That would mean that we initialize the Symbolizer after we free the raw profile data buffer.

Reply:
Yeah that's a good idea. In fact, we can move it after setupForSymbolization(), although I don't think there is a memory savings from doing that.